### PR TITLE
[chrome] Add `PlatformInfo` enums

### PR DESCRIPTION
**Pull request by franciscocpg**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -6534,7 +6536,7 @@ declare namespace chrome.runtime {
          * The operating system chrome is running on.
          * One of: "mac", "win", "android", "cros", "linux", or "openbsd"
```
Comment (nicolas377) [inline types/chrome/index.d.ts:6537]:
```suggestion
         * The operating system chrome is running on.
```

Comment (nicolas377) [inline types/chrome/index.d.ts:6537]:
```suggestion
         * The operating system chrome is running on.
```

**Review by nicolas377 — COMMENTED** [review]
Hey @franciscocpg! You should probably edit some tests for this change. It'll make merging this a lot easier, and this type of changes should have test changes anyways!

**Comment by franciscocpg** [comment]
> Hey @franciscocpg! You should probably edit some tests for this change. It'll make merging this a lot easier, and this type of changes should have test changes anyways!

Hi @nicolas377.

After taking a look at the [docs](https://developer.chrome.com/docs/extensions/reference/runtime/#type-PlatformInfo) I decided to implement all the enums used in `PlatformInfo`. I also added tests.

**Review by nicolas377 — APPROVED** [review]
All good! I tested this out in the Playground, and it works fine!

**Comment by franciscocpg** [comment]
Ready to merge

**Pull request merged**